lisp/c/matrix.c

```diff
@@ -1103,14 +1103,14 @@ pointer argv[];
     printf("rotation-angle1: %f %f %f\n",kx,ky,kz);
   if ((fabs(kx) > fabs(ky)) && (fabs(kx) > fabs(kz))) {
     ky=(m[size]+m[1])/(2*kx*vers); kz=(m[2]+m[size+size])/(2*kx*vers);
-    norm=sqrt((ky*ky+kz*kz)/(1.0-kx*kx)); ky/=norm; kz/=norm;}
+    norm=sqrt((ky*ky+kz*kz)/(1.0-kx*kx)); if (!isnan(norm)) {ky/=norm; kz/=norm;}}
```

Comment:
What happens for the `else` case?

Reply:
This line tries to normalize axis vector. In my understanding, axis vector calculated before this line is already normalized, so https://github.com/euslisp/EusLisp/blob/master/lisp/c/matrix.c#L1101-L1114 region code actually do nothing. But I don't completely understand this code, I only add nan check instead of removing whole of this line.